src/components/BookList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookList = ({ book, removeBookProp }) => (
+  <div key={book.id} className="bookRow">
+    <div className="bookDisplay">
+      <h3>{book.title}</h3>
+      <p>
+        {book.author}
+      </p>
+    </div>
+    <button type="submit" className="removeBtn" onClick={() => removeBookProp(book.id)}>Remove</button>
+  </div>
+);
+
+BookList.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
- [ ] Kindly remove the eslint disable rule. It's best to fix the issue in the code.